apps/meteor/server/routes/avatar/middlewares/index.js

```diff
@@ -5,3 +5,4 @@ import { protectAvatars } from './auth';
 import './browserVersion';
 
 WebApp.connectHandlers.use('/avatar/', protectAvatars);
+WebApp.connectHandlers.use('/avatar/uid/', protectAvatars);
```

Comment:
For users without permission to access the avatar, this will treat the uid as an username and render its first digit as an avatar. I don't know if that is an OK behavior?

Reply:
I made another function for this specific case. Thanks for pointing this out, I took the function name at face value.